MinimedKit/Models/PumpModel.swift

```diff
@@ -51,7 +51,8 @@ public enum PumpModel: String {
     }
     
     public var hasMySentry: Bool {
-        return generation >= 23
+        //TODO: disable MySentry when using OL return generation >= 23
```

Comment:
This needs to be tackled at a higher level. PumpModels >= 23 should still return that they have MySentry capability, even when using OL, but the MinimedPumpManager could choose not to use it if using OL.

Reply:
There is a "Use MySentry" option in pump settings in dev that can be used to enable/disable use of MySentry. The battery tradeoffs are similar on both RL and OL.